code/game/runtimes.dm

```diff
@@ -2,33 +2,41 @@
 	Custom runtime handling
 */
 
+// Early errors handling:
+//  For all these cases were errors might occur before logging/debugguer is ready, we stash them away
+//  Can't trust static initializers here so default/values must be handled at runtime
+//  Do NOT convert these to GLOB because errors can happen BEFORE GLOB exists,
+//  which would cause /world/Error handler to also crash and make them all silent!
 
-// /world/Error might be called during early static init, in which case we don't have
-// GLOB and STUI facilities yet. Initializer expressions for the following globals may not
-// even have been ran yet! In such a case we just log them to globals and let
-// MC-init-time SSearlyruntimes pick them up.
+var/list/stui_init_runtimes //! Shorthand of Static Initializer errors only, for use in STUI
+var/list/full_init_runtimes //! Full text of all Static Initializer + World Init errors, for log backfilling
+var/runtime_logging_ready //! Truthy when init is done and we don't need these shenanigans anymore
+var/init_runtimes_count //! Count of runtimes that occured before logging is ready, for in-game reporting
```

Comment:
what's wrong with using the GLOBAL_REAL macros here?

Reply:
I think I removed it at some point for consistency because they don't allow initializers, but in the end you can't use the initializers anyway here... Gonna move them back to GLOBAL_REAL format